wcfsetup/install/files/lib/system/worker/SitemapRebuildWorker.class.php

```diff
@@ -421,4 +429,25 @@ private function changeUserToGuest() {
 	private function changeToActualUser() {
 		WCF::getSession()->changeUser($this->actualUser, true);
 	}
+	
+	/**
+	 * Reads the columns changed by the user for this sitemap object from the registry and modifies the object accordingly.
+	 * 
+	 * @param       ObjectType      $object
+	 * @since       5.3
+	 */
+	public static function prepareSitemapObject(ObjectType &$object) {
```

Comment:
```suggestion
	public static function prepareSitemapObject(ObjectType $object) {
```
Objects are always passed reference by value. No need to pass the reference by reference.